src/plugins/intel_cpu/src/transformations/snippets/x64/op/brgemm_utils.cpp

```diff
@@ -83,6 +85,20 @@ size_t compute_inner_n_block(const ov::element::Type& precision) {
         default: OPENVINO_THROW("BrgemmCopyB doesn't support precision ", precision);
     }
 }
+
+const ov::snippets::lowered::ExpressionPtr get_copy_b_expr(const ov::snippets::lowered::ExpressionPtr& brgemm_expr) {
+    OPENVINO_ASSERT(ov::is_type<BrgemmCPU>(brgemm_expr->get_node()), "get_copy_b_expr must be called only for BrgemmCPU node");
+    const auto b_input_expr = brgemm_expr->get_input_port_connector(1)->get_source().get_expr();
+    if (ov::is_type<BrgemmCopyB>(b_input_expr->get_node())) {
+        return b_input_expr;
+    } else if (ov::is_type<snippets::op::Buffer>(b_input_expr->get_node())) {
+        const auto input_buffer_expr = b_input_expr->get_input_port_connector(0)->get_source().get_expr();
```

Comment:
Suggest to add assert that `BufferExpression` has one input at least here

Reply:
Done, thanks